src/main/java/bio/terra/app/utils/ControllerUtils.java

```diff
@@ -38,4 +41,19 @@ public static void validateEnumerateParams(Integer offset, Integer limit, String
     public static void validateEnumerateParams(Integer offset, Integer limit) {
         validateEnumerateParams(offset, limit, null, null);
     }
+
+    public static ResponseEntity<JobModel> jobToResponse(JobModel job) {
```

Comment:
Not sure how jade decides what goes where with respect to static methods, but maybe a better place for this is `JobService`, which already has code that deals with `JobModel`?

Reply:
Kind of a toss up. The controller is where we deal with (almost) all thing ResponseEntity.